force-app/main/default/classes/IBMNaturalLanguageClassifierV1Test.cls

```diff
@@ -0,0 +1,171 @@
+@isTest
+private class IBMNaturalLanguageClassifierV1Test {
+
+  /**
+   *  Test phrase to classify.
+   *
+   */
+  static testMethod void testClassify() {
+    String body = IBMWatsonMockResponses.Classify();
+    IBMWatsonMockHttpResponse mockResponse = new IBMWatsonMockHttpResponse(200, 'Success', body, null);
+    Test.setMock(HttpCalloutMock.class, mockResponse);
+    Test.startTest();
+    IBMNaturalLanguageClassifierV1 service = new IBMNaturalLanguageClassifierV1();
+    service.setEndPoint('https://gateway.watsonplatform.net/discovery/api');
+    service.setUsernameAndPassword('username', 'password');
+    String classifier_id_serialized_name = '10D41B-nlc-1'; 
+    String text_serialized_name = 'How hot will it be today?';
+    // this would be the expected top class as defined on the mock response
+    String top_class_serialized_name = 'temperature';
+    IBMNaturalLanguageClassifierV1Models.ClassifyOptions classifyOptions = new IBMNaturalLanguageClassifierV1Models.ClassifyOptionsBuilder()
+      .classifierId(classifier_id_serialized_name)
+      .text(text_serialized_name)
+      .build();
+    IBMNaturalLanguageClassifierV1Models.Classification classification = service.classify(classifyOptions);
+    System.assertEquals(classification.getClassifierId(), classifier_id_serialized_name);
+    System.assertEquals(classification.getUrl(), IBMWatsonMockResponses.IBMNaturalLanguageClassifierClassifyURL);
+    System.assertEquals(classification.getText(), text_serialized_name);
+    System.assertEquals(classification.getTopClass(), top_class_serialized_name);
+    List<IBMNaturalLanguageClassifierV1Models.ClassifiedClass> ClassifiedClassList = classification.getClasses();
+    System.assertNotEquals(ClassifiedClassList, null);
+    System.assertEquals(ClassifiedClassList.size(), 2);
+    IBMNaturalLanguageClassifierV1Models.ClassifiedClass ClassifiedClass = ClassifiedClassList[0];
+    String class_name_serialized_name = ClassifiedClass.getClassName();
+    Set<String> class_name_set = new Set<String>{'temperature', 'conditions'};
+    System.assert(class_name_set.contains(class_name_serialized_name), 'Invalid class name for classifier.');
+    Double confidence_serialized_name = ClassifiedClass.getConfidence();
+    System.assertNotEquals(confidence_serialized_name, null);
+    System.assert(confidence_serialized_name>0, 'Invalid confidence for classifier.');
```

Comment:
Use
```apex
System.assert(confidence_serialized_name > 0, 'Invalid confidence for classifier.');
```
Instead of 
```apex
System.assert(confidence_serialized_name>0, 'Invalid confidence for classifier.');
```